src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionCouponsContext.php

```diff
@@ -239,7 +257,7 @@ public function iDeleteThem(): void
     }
 
     /**
-     * @Then /^there should be (\d+) coupon related to (this promotion)$/
+     * @Then /^there should be (\d+) coupon(?:|s) related to (this promotion)$/
```

Comment:
```suggestion
     * @Then /^there should be 1 coupon related to (this promotion)$/
     * @Then /^there should be (\d+) coupons related to (this promotion)$/
```

I know it won't work probably, but do you see what I'd like to achieve? :D
Forbid the `Then there should be 5 coupon...` usage in scenarios :D